### PR TITLE
Feature/notifications libraries

### PR DESCRIPTION
**Pull request by iariasleon**

### CODE REVIEW

File: test/acceptance/behave/README.md [hunk 1]
```diff
@@ -228,6 +228,13 @@ Feature: feature name...
 ```
 
 
+## Notification listener
+
+the notification listener is executed automatically (a HTTP server as a daemon) in local IP where test are executed.
```
Comment (fgalan) [inline test/acceptance/behave/README.md]: the -> The

File: test/acceptance/behave/tools/notification_listener.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    if VERBOSE:
+        __logger__.debug("Request data received in listener:\n Url: %s\n Headers: %s\n Payload: %s\n" % (last_url, str(last_headers), last_payload))
+
+class MyHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+    """
+    HTTP server
+    """
+    def do_POST(s):
+        """
+        POST request
```
Comment (fgalan) [inline test/acceptance/behave/tools/notification_listener.py]: :param s: missing

File: test/acceptance/behave/tools/notification_listener.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        for item in last_headers:
+            s.send_header("%s-%s" % (headers_prefix, item), last_headers[item])
+        s.send_header("%s-url" % headers_prefix, "%s" % (last_url))
+
+        show_last_data()   # verify VERBOSE global variable
+
+        s.end_headers()
+        s.wfile.write(last_payload)
+
+    def do_GET(s):
```
Comment (fgalan) [inline test/acceptance/behave/tools/notification_listener.py]: :param s: missing

Comment (iariasleon) [inline test/acceptance/behave/tools/notification_listener.py]: `s` is replaced to `self`
FIxed in  	a65d8bd

File: test/acceptance/behave/tools/notification_listener.py (hunk 2 above)
Comment (iariasleon) [inline test/acceptance/behave/tools/notification_listener.py]: `s` is replaced to `self`
FIxed in  	a65d8bd

File: test/acceptance/behave/README.md (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/behave/README.md]: FIxed in  	a65d8bd

**Comment by fgalan** [comment]
LGTM

**Pull request merged**